proposals/2024-01-29_native_histograms_text_format.md

```diff
@@ -0,0 +1,90 @@
+## Native Histograms Text Format
+
+* **Owners:**
+  * Chris Marchbanks [@csmarchbanks](https://github.com/csmarchbanks)
+
+* **Implementation Status:** Not implemented
+
+* **Related Issues and PRs:**
+  * [Native Histogram Support in client_python](https://github.com/prometheus/client_python/issues/918)
+  * [OpenMetrics Protobuf format PR](https://github.com/OpenObservability/OpenMetrics/pull/256)
+
+* **Other docs or links:**
+  * [Design Doc for choosing a proposal](https://docs.google.com/document/d/1qoHf24cKMpa1QHskIjgzyf3oFhIPvacyJj8Tbe6fIrY/edit#heading=h.5sybau7waq2q)
+
+> TL;DR: This design doc is proposing a format for exposing native histograms in the OpenMetrics text format.
+
+## Why
+
+Today it is only possible to export native histograms using the Protocol Buffers (protobuf) scrape format. Many users prefer the text format, and some client libraries, such as the Python client, want to avoid adding a dependency on protobuf. 
```

Comment:
To avoid confusion, can we clarify that the protobuf format used isn't even the OpenMetrics protobuf format, but the classic Prometheus protobuf format. (The proposed changes to support native histograms in the OpenMetrics protobuf format is linked above, so here we could explain that.)